Source/Diagnostics/ReducedDiags/FieldMomentum.H

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2019-2020
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#ifndef WARPX_DIAGNOSTICS_REDUCEDDIAGS_FIELDMOMENTUM_H_
+#define WARPX_DIAGNOSTICS_REDUCEDDIAGS_FIELDMOMENTUM_H_
+
+#include "ReducedDiags.H"
+#include <fstream>
```

Comment:
```suggestion
```

Don't think you need that include. And if we want to be very exhaustive with includes, I guess you could add `#include <string>` instead (although being exhaustive with includes looks hard and tedious after looking at #1947 :grin: )